tests/unit/custom_fee_test.py

```
@@ -156,59 +163,68 @@ def test_custom_royalty_fee():
-        "custom_royalty_fee, expected_str",
-        [
-            (
-                CustomRoyaltyFee(
-                   numerator=3,
-                   denominator=20,
-                   fallback_fee=None,
-                   fee_collector_account_id=None,
-                   all_collectors_are_exempt=True,
-                ),
-                "\n".join([
+    "custom_royalty_fee, expected_str",
```

Ruff (0.14.14)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)